wolfssl/openssl/asn1.h

```diff
@@ -16,4 +17,22 @@ struct WOLFSSL_ASN1_STRING {
     long flags;
 };
 
+
+WOLFSSL_API long wolfSSL_ASN1_INTEGER_get(const WOLFSSL_ASN1_INTEGER* i);
+WOLFSSL_API int wolfSSL_ASN1_INTEGER_get_int64(signed_word64 *p, const WOLFSSL_ASN1_INTEGER *i);
+
+#define ASN1_INTEGER WOLFSSL_ASN1_INTEGER
```

Comment:
As far as I checked, "typedef WOLFSSL_ASN1_INTEGER ASN1_INTEGER;" appears only once in <wolfssl/openssl/ssl.h> in the whole wolfssl project. I'm wondering why in your case.